src/fileseq/filesequence.py

```diff
@@ -62,7 +62,11 @@ def __init__(self, sequence):
         else:
             self._dir = ''
 
-        self._zfill = sum([PAD_MAP[c] for c in self._pad])
+        printf_len = PRINTF_RE.split(self._pad, 1)
+        if len(printf_len) == 3:
```

Comment:
I had to stare at this a bit and even try it out in a repl to realize what the significance of `3` was. This seems to be a  more obscure way to represent it. Do you think it would be more readable to do this?

```python
match = PRINTF_RE.match(self._pad)
if match:
    self._zfill = match.group(1)
```